src/NovelRT/CMakeLists.txt

```diff
@@ -89,7 +89,7 @@ set(SOURCES
 
 add_library(Engine SHARED ${SOURCES})
 
-if(CMAKE_BUILD_TYPE MATCHES RELEASE)
+if(CMAKE_BUILD_TYPE STREQUAL "Release")
   set_property(TARGET Engine PROPERTY OUTPUT_NAME "NovelRT")
 else()
   set_property(TARGET Engine PROPERTY OUTPUT_NAME "NovelRT-$<CONFIG>")
```

Comment:
Hey @RubyNova - it looks like macOS and Ubuntu are fine with this, but Windows is still outputting a `NovelRT-Release.dll` file instead of `NovelRT.dll`.

Any suggestions on how to make Windows respect this, @FiniteReality ?